ios/Exponent.xcodeproj/project.pbxproj

```diff
@@ -2225,6 +2229,7 @@
 						CreatedOnToolsVersion = 6.2;
 						DevelopmentTeam = C8D8QTF339;
 						ProvisioningStyle = Automatic;
+						ProvisioningStyle = Manual;
```

Comment:
I'm not an expert and it could be just `Xcode things`, but this fragment seems suspicious to me (`ProvisioningStyle = a`, `ProvisioningStyle = b`)

Reply:
Good catch. I had to change the provisioning style to manual in order to get uploading a custom version of the Client to work because it required special permissions that we typically don't include to accept payments. Will have to change back to automatic before merge.